src/lib/divisions.ml

```diff
@@ -0,0 +1,70 @@
+
+(* This file is free software, part of FTW. See file "LICENSE" for more information *)
+
+(* Type definitions *)
```

Comment:
Add `Divisions` concept to `concepts.md` ?

Reply:
Done